examples/template/spreadsheet_builder_constants.py

```diff
@@ -0,0 +1,177 @@
+                       '/collection_protocol',
+                       'https://schema.humancellatlas.org/type/protocol/biomaterial_collection/2.0.0'
+                       '/aggregate_generation_protocol',
+                       'https://schema.humancellatlas.org/type/protocol/biomaterial/5.1.0'
+                       '/biomaterial_collection_protocol',
+                       'https://schema.humancellatlas.org/type/protocol/analysis/9.0.0/analysis_protocol',
+                       'https://schema.humancellatlas.org/type/protocol/7.0.0/protocol',
+                       'https://schema.humancellatlas.org/type/project/14.0.0/project',
+                       'https://schema.humancellatlas.org/type/process/sequencing/5.1.0/sequencing_process',
+                       'https://schema.humancellatlas.org/type/process/sequencing/5.1.0/library_preparation_process',
+                       'https://schema.humancellatlas.org/type/process/imaging/5.1.0/imaging_process',
+                       'https://schema.humancellatlas.org/type/process/biomaterial_collection/5.1.0/enrichment_process',
+                       'https://schema.humancellatlas.org/type/process/biomaterial_collection/5.1.0'
+                       '/dissociation_process',
+                       'https://schema.humancellatlas.org/type/process/biomaterial_collection/5.1.0/collection_process',
+                       'https://schema.humancellatlas.org/type/process/analysis/11.0.1/analysis_process',
+                       'https://schema.humancellatlas.org/type/process/9.1.0/process',
+                       'https://schema.humancellatlas.org/type/file/2.0.0/supplementary_file',
+                       'https://schema.humancellatlas.org/type/file/9.0.0/sequence_file',
+                       'https://schema.humancellatlas.org/type/file/3.0.0/reference_file',
+                       'https://schema.humancellatlas.org/type/file/2.0.0/image_file',
+                       'https://schema.humancellatlas.org/type/file/6.0.0/analysis_file',
+                       'https://schema.humancellatlas.org/type/biomaterial/10.2.0/specimen_from_organism',
+                       'https://schema.humancellatlas.org/type/biomaterial/11.1.0/organoid',
+                       'https://schema.humancellatlas.org/type/biomaterial/3.1.0/imaged_specimen',
+                       'https://schema.humancellatlas.org/type/biomaterial/15.3.0/donor_organism',
+                       'https://schema.humancellatlas.org/type/biomaterial/13.1.0/cell_suspension',
+                       'https://schema.humancellatlas.org/type/biomaterial/14.3.0/cell_line']
+
+"""
+Two additional pieces of information must be provided to a LinkedSpreadsheetBuilder in order to enable linking fields 
+between metadata schema in the spreadsheet (i.e. copying over a column from one tab to another tab).
+
+- Make one multitab spreadsheet per backbone (this is not a list as initally discussed envisioned)
```

Comment:
Initially discussed by whom? Probably not right to put this in code, instead in a github issue? (This may apply to this entire list)

Reply:
Covered by the improvements to documentation/docstring ticket I filed: https://app.zenhub.com/workspaces/dcp-5ac7bcf9465cb172b77760d9/issues/humancellatlas/ingest-client/38. Also added a TODO in the code, assigning myself.